src/redis/commands.cr

```diff
@@ -134,6 +134,19 @@ class Redis
       integer_command(concat(["DEL"], keys))
     end
 
+    # Removes the specified keys.
+    #
+    # **Return value**: Integer, the number of keys that were removed.
+    #
+    # Example:
+    #
+    # ```
+    # redis.del(["some", "keys", "to", "delete"])
+    # ```
+    def del(keys : Array)
```

Comment:
In hindsight, I think this should have been the API, not the current arguments splat.